src/compute-types/src/dyncfgs.rs

```diff
@@ -229,6 +229,13 @@ pub const COMPUTE_LOGICAL_BACKPRESSURE_INFLIGHT_SLACK: Config<Duration> = Config
     "Round observed timestamps to slack.",
 );
 
+/// Enable per-column dictionary compression for row containers in arrangements.
+pub const ENABLE_ARRANGEMENT_DICTIONARY_COMPRESSION: Config<bool> = Config::new(
+    "enable_arrangement_dictionary_compression",
+    true,
```

Comment:
```suggestion
    false,
```

Please disable by default before you merge.